### PR TITLE
Update react and eslint

### PR DESCRIPTION
**Pull request by katerinamartysko**

### CODE REVIEW

**Comment by katerinamartysko** [comment]
[Update react and ESLint](https://trello.com/c/yGNcWV6N/4-update-react-and-eslint)

File: .eslintrc.js [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "ecmaVersion": "latest",
+        "sourceType": "module"
+    },
+    "plugins": [
+        "react",
+        "@typescript-eslint"
+    ],
+    "rules": {
```
Comment (LexaLukovka) [inline .eslintrc.js]: Please remove the empty `rules` object

File: .eslintrc.js [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
+    "env": {
+        "browser": true,
+        "es2021": true
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:react/recommended",
+        "plugin:@typescript-eslint/recommended"
+    ],
+    "overrides": [
```
Comment (LexaLukovka) [inline .eslintrc.js]: Please remove the empty `overrides` array

File: .eslintrc.js (hunk 1 above)
Comment (katerinamartysko) [reply .eslintrc.js]: Done

File: .eslintrc.js (hunk 2 above)
Comment (katerinamartysko) [reply .eslintrc.js]: Done

**Review by LexaLukovka — APPROVED** [review]

**Pull request merged**